templates/CRM/UF/Form/Group.hlp

```diff
@@ -114,7 +114,7 @@
 {htxt id='id-add_captcha'}
 {capture assign="miscURL"}{crmURL p="civicrm/admin/setting/recaptcha" q="reset=1"}{/capture}
 <p>{ts}When reCAPTCHA is enabled for a profile form, anonymous users are required to read an image with letters and numbers and enter the value in a field. This helps prevent abuse by automated scripts.{/ts}</p>
-<p>{ts 1="https://www.google.com/recaptcha" 2=$miscURL}To use reCAPTCHA you must sign up at <a href="%1" target="_blank">Google's reCaptcha site</a> to get your public and private keys. Then enter both keys in <a href="%2">Administer CiviCRM &raquo; Customize Data and Screens &raquo; reCAPTCHA settings</a>.{/ts}</p>
+<p>{ts 1="https://www.google.com/recaptcha" 2=$miscURL}To use reCAPTCHA you must sign up at <a href="%1" target="_blank">Google's reCaptcha site</a> to get your public and private keys. Then enter both keys in <a href="%2">Administer CiviCRM &raquo; System Settings &raquo; reCAPTCHA settings</a>.{/ts}</p>
```

Comment:
Since it's changing the string anyway, is it worth "fixing" the `<a href="%1" target="_blank">` to `<a %1>`? Ditto %2 ?

Reply:
Good idea - updated the PR.